mbs-s0003-20230501/README.md

```diff
@@ -0,0 +1,140 @@
+CDELT1  =   -0.033333333333333 / [deg] Coordinate increment at reference point  
+CDELT2  =    0.033333333333333 / [deg] Coordinate increment at reference point  
+CUNIT1  = 'deg'                / Units of coordinate increment and value        
+CUNIT2  = 'deg'                / Units of coordinate increment and value        
+CTYPE1  = 'RA---CAR'           / Right ascension, plate caree projection        
+CTYPE2  = 'DEC--CAR'           / Declination, plate caree projection            
+CRVAL1  =                  0.0 / [deg] Coordinate value at reference point      
+CRVAL2  =                  0.0 / [deg] Coordinate value at reference point      
+LONPOLE =                  0.0 / [deg] Native longitude of celestial pole       
+LATPOLE =                 90.0 / [deg] Native latitude of celestial pole        
+MJDREF  =                  0.0 / [d] MJD of fiducial time                       
+RADESYS = 'ICRS'               / Equatorial coordinate system                   END
+```
+
+## Example usage
+
+We give some minimum working examples:
+```python
+import numpy as np
+from pixell import enmap, curvesky
+from os.path import join as opj
+
+# first get some basic info like the path and the filename template
+fdir = '/global/cfs/cdirs/sobs/v4_sims/mbs/mbs-s0003-20230501/sims'
+fbase_template = 'so_scan_s0003_fdw_{bands}_lmax5400_4way_set{split_num}_noise_sim_map{sim_num}.fits'
+
+# let's load the sim for e.g. f150, split 2, sim 123
+bands = 'mf_f090_mf_f150'
+band_idx = 1                # f090 is 0, f150 is 1
+split_num = 2
+sim_num = str(123).zfill(4) # has 4 digits (leading 0's)
+
+sim = enmap.read_map(
+    opj(fdir, fbase_template.format(
+        bands=bands,
+        split_num=split_num,
+        sim_num=sim_num)
+        ),
+    sel=np.s_[band_idx, 0]  # just load f150, remove split axis
+    )
+print(sim.geometry, sim.dtype)
+```
+This should give:
+```python
+((3, 2640, 10800), car:{cdelt:[-0.03333,0.03333],crval:[0,0],crpix:[5400.62,1890.50]}) float32
+```
+As discussed, the `sim` only contains noise power to the Nyquist frequency of the pixelization (for 2 arcmin pixels, `lmax=5400`). But what if we wanted to work with objects in map-space defined at the full resolution of the data, for example a sky mask defined at 0.5 arcmin resolution? We need to project the `sim` to our desired geometry.
+
+Because the `sim` is bandlimited, we can do this losslessly with a spherical harmonic transforms:
+```python
+# first we need the geometry of the pixelization.
+# we can get this from one of the time-domain simulations
+shape, wcs = enmap.read_map_geometry('/global/cfs/cdirs/sobs/sims/scan-s0003/output/combined/processed/fejer1/lat01_s25_fullfp_f150_1pass_4way_set02_map.fits')
+
+# allocate an empty map to hold the sim at full resolution
+sim_fullres = enmap.empty((*sim.shape[:-2], *shape[-2:]), wcs, sim.dtype)
+
+# project sim to alm, then from alm to map
+curvedsky.alm2map(curvedsky.map2alm(sim, lmax=5400), sim_fullres)
```

Comment:
I will look at this code more carefully later on.

Reply:
Ok